.travis.yml

```diff
@@ -8,6 +8,10 @@ rvm:
   - 2.5
```

Comment:
I dont think we need anything before 2.7 since they are EOL

Reply:
2.7 eol too, I say delete travis if we do anything with it

Reply:
Yeah let's drop travis

Reply:
DOWN WITH TRAVIS!!